src/structures/interfaces/TextBasedChannel.js

```diff
@@ -58,6 +58,7 @@ class TextBasedChannel {
    * @property {string} [content=''] The content for the message
    * @property {MessageEmbed|Object} [embed] An embed for the message
    * (see [here](https://discordapp.com/developers/docs/resources/channel#embed-object) for more details)
+   * @property {MessageMentionOptions} [allowedMentions] Which mentions should be parsed from the message content
```

Comment:
shouldn't there be a jsdoc typedef for `MessageMentionOptions`?

Reply:
Yes, done